imports/client/components/Routes.tsx

```diff
@@ -1,69 +1,45 @@
-            {/* Authenticated routes */}
-            {/*
-             // @ts-ignore (hopefully this will go away with react-router v5 anyway) */}
-            <Route path="/" component={Authenticator} authenticated>
-              <IndexRedirect to="hunts" />
-              <Route path="" component={App}>
-                <Route path="hunts/:huntId" component={HuntApp}>
-                  <Route path="announcements" component={AnnouncementsPage} />
-                  <Route path="guesses" component={GuessQueuePage} />
-                  <Route path="hunters" component={HuntProfileListPage} />
-                  <Route path="hunters/invite" component={UserInvitePage} />
-                  <Route path="puzzles/:puzzleId" component={PuzzlePage} />
-                  <Route path="puzzles" component={PuzzleListPage} />
-                  <IndexRedirect to="puzzles" />
-                </Route>
-                <Route path="hunts" component={HuntListPage} />
-                <Route path="users/:userId" component={ProfilePage} />
-                <Route path="users" component={AllProfileListPage} />
-                <Route path="setup" component={SetupPage} />
-              </Route>
-            </Route>
-            {/* Unauthenticated routes */}
-            {/*
-             // @ts-ignore (hopefully this will go away with react-router v5 anyway) */}
-            <Route path="/" component={Authenticator} authenticated={false}>
-              <Route path="" component={SplashPage}>
-                <Route path="login" component={LoginForm} />
-                <Route path="reset-password/:token" component={PasswordResetForm} />
-                <Route path="enroll/:token" component={EnrollForm} />
-              </Route>
-            </Route>
-            {/* Routes available to both authenticated and unauthenticated users */}
-          </Router>
+          <BrowserRouter>
+            <Switch>
+              {/* Index redirect */}
+              <Route exact path="/" component={RootRedirector} />
+
+              {/* Authenticated routes - if user not logged in, get redirected to /login */}
+              <AuthenticatedRoute path="/hunts/:huntId" component={HuntApp} />
+              <AuthenticatedRoute path="/hunts" component={HuntListPage} />
+              <AuthenticatedRoute path="/users/:userId" component={ProfilePage} />
+              <AuthenticatedRoute path="/users" component={AllProfileListPage} />
+              <AuthenticatedRoute path="/setup" component={SetupPage} />
+
+              {/* Unauthenticated routes - if user already logged in, get redirected to /hunts */}
+              <UnauthenticatedRoute path="/login" component={LoginForm} />
+              <UnauthenticatedRoute path="/reset-password/:token" component={PasswordResetForm} />
+              <UnauthenticatedRoute path="/enroll/:token" component={EnrollForm} />
+
+              {/* Routes available to both authenticated and unauthenticated users */}
```

Comment:
I feel like maybe we should just kill this comment since it doesn't seem like we're ever going to have any of these

Reply:
Done.